_config.yml

```diff
@@ -11,23 +11,23 @@
 # you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: LinuxChix India
-email: indichix@linuxchix.org # Mailing list
+title: Women Who Go Delhi 
+email: delhi@womenwhogo.org # Mailing list
 description: > # this means to ignore newlines until "baseurl:"
-  LinuxChix is a community for women who like Linux and for anyone who wants to support women in computing. 
-  We aim to promote FOSS primarily Linux and related projects among women and other minorities in tech. 
-  We are a group of enthusiastic people ranging from students to professional experts willing to teach and foster the growth of Linux.
-baseurl: "/" # the subpath of your site, e.g. /blog
+  You do not need to know Go, be a professional developer or enjoy talking to strangers in order to come hang out with us!
+  This group is for you if you are interested in Go , and are looking for a safe space to learn.
+
+baseurl: "" # the subpath of your site, e.g. /blog
```

Comment:
Could you please add the `baseurl` back? 
Because it's not going to work once pushed upstream.